go/src/koding/klientctl/config/config.go

```diff
@@ -82,6 +82,11 @@ var (
 	// KontrolURL is overwritten during deploy via linker flag.
 	KontrolURL = "https://koding.com/kontrol/kite"
 
+	// TunnelKiteAddress is the address that koding's tunnel service is run on.
+	//
+	// This is overwritten during deploy via linker flag.
+	TunnelKiteAddress = "http://t.koding.com/kite"
```

Comment:
I was too fast, now I noticed we overwrite the string during compile. Well, the long term plan is to get rid of injecting configuration string during compile in order to have only 1 klient channel, not 4 (less things to compile and deploy).

For now we must live with the injection during compile time.